### PR TITLE
Update captions 

### PR DESCRIPTION
**Pull request by Chrispy03**
related to #2 

TL;DR: changing captions for slide deck

### CODE REVIEW

File: _posts/0000-01-15-Chrispy03.md [hunk 1]
```diff
@@ -3,7 +3,8 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
+
 ![winter-is-coming](https://cloud.githubusercontent.com/assets/16547949/25401218/203cc85a-29c3-11e7-8bbf-8c7d4a6b81b2.jpg)
 {: .center}
 
-Caption here
+Hello World whatever
```
Comment (Chrispy03) [inline _posts/0000-01-15-Chrispy03.md]: what does this mean

Comment (hammj) [inline _posts/0000-01-15-Chrispy03.md]: We should probably run this by the Env Supervisor

Comment (hammj) [inline _posts/0000-01-15-Chrispy03.md]: Chris said this was as good as it gets.

**Comment by Chrispy03** [comment]
done with editing for now i guess

**Pull request closed without merging**